tests/unit-core/test-api.c

```diff
@@ -161,8 +161,7 @@ handler_construct (const jerry_call_info_t *call_info_p, /**< call information *
   TEST_ASSERT (jerry_value_is_object (call_info_p->this_value));
 
   TEST_ASSERT (args_cnt == 1);
-  TEST_ASSERT (jerry_value_is_boolean (args_p[0]));
-  TEST_ASSERT (jerry_get_boolean_value (args_p[0]) == true);
+  TEST_ASSERT (jerry_value_is_true(args_p[0]) == true);
```

Comment:
Space before `(`. Similar cases below.

Reply:
Btw, I think `TEST_ASSERT (jerry_value_is_true (args_p[0]));` is enough since we test a condition, not an expected value (What we did for `jerry_get_boolean_value`)